extensions/wikia/Recirculation/js/helpers/LiftigniterHelper.js

```diff
@@ -5,95 +5,116 @@ define('ext.wikia.recirculation.helpers.liftigniter', [
 ], function ($, w, thumbnailer) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "thumbnailer". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AUnusedFunctionArgument)